apps/client/src/hooks/application/use-application.ts

```diff
@@ -0,0 +1,33 @@
+import { useNavigate } from 'react-router-dom';
+import { useDismissAllNotifications } from '~/hooks/notifications/use-dismiss-all-notifications';
+import { useSetNavigationVisibility } from './use-set-navigation-visibility';
+
+/** Use to control the application. */
+export function useApplication() {
```

Comment:
should this be `useNavigation`? `useApplication` seems maybe a little broad :).

Reply:
Will fix it forward!